src/utility.c

```diff
@@ -811,7 +817,20 @@ int colour_str_to_colour(char *colour_str) {
     } else if (strcmp(colour_str, "ora") == 0) {
         colour = STRING_ORA;
     } else if (strcmp(colour_str, "ran") == 0) {
-        colour = (int)(((float)rand() / (float)RAND_MAX) * (float)WHITE);
+        /* prevent divide by zero */
+        if (ran_target_fps < 1) {
+            ran_target_fps = 1;
+        }
+        if (ran_target_fps < 50) {
+            /* limit the framerate if desired */
+            if ((get_ticks() - last_random_colour) >= (1000 / ran_target_fps)) {
+                random_colour = (int)(((float)rand() / (float)RAND_MAX) * (float)WHITE);
+                last_random_colour = get_ticks();
```

Comment:
this variable should be called something like `last_colour_ticks` or something since it's not storing a colour